test/fixtures/node.yml

```diff
@@ -6,7 +6,6 @@ one:
   status: 1
   type: "note"
   uid: 2
-  nid: 1
```

Comment:
I changed line 52 of notes_controller_test.rb to `note = DrupalNode.where(type: 'note', status: 0).first`, which solved one issue, so we're not calling a fixture by id. But now it doesn't have a node_counter, so it fails to increment the views count, line 342 of models/drupal_node.rb:

    self.drupal_node_counter.totalcount += 1                                                                                                                        

This requires the fixture in node_counter.yml to associate. I'm using the 

    node: one

syntax you used to try to fix this.